hyundai_canfd.dbc

```diff
@@ -563,6 +568,7 @@ CM_ SG_ 1043 COUNTER_ALT "only increments on change";
 VAL_ 53 GEAR 0 "P" 5 "D" 6 "N" 7 "R" ;
 VAL_ 64 GEAR 0 "P" 5 "D" 6 "N" 7 "R" ;
 VAL_ 69 GEAR 0 "P" 5 "D" 6 "N" 7 "R" ;
+VAL_ 112 GEAR 0 "P" 5 "D" 6 "N" 7 "R" ;
```

Comment:
I'm not sure this is right, in this route it's saying 4 is drive: https://cabana.comma.ai/?route=db04d2c63990e3ba%7C2022-12-21--15-25-04&seekTime=14&url=https%3A%2F%2Fchffrprivate.azureedge.net%2Fchffrprivate3%2Fv2%2Fdb04d2c63990e3ba%2F86251558a0e6b7d20794b1dd3a57377d_2022-12-21--15-25-04

Reply:
> I'm not sure this is right, in this route it's saying 4 is drive: https://cabana.comma.ai/?route=db04d2c63990e3ba%7C2022-12-21--15-25-04&seekTime=14&url=https%3A%2F%2Fchffrprivate.azureedge.net%2Fchffrprivate3%2Fv2%2Fdb04d2c63990e3ba%2F86251558a0e6b7d20794b1dd3a57377d_2022-12-21--15-25-04

@sshane  It's showing up as 5 is drive on my end in this route. Maybe a bug with cabana?
![image](https://user-images.githubusercontent.com/47793918/210594725-6785ccca-0555-4d89-92fb-8b8afcd909a1.png)